CHANGES.md

```diff
@@ -1,3 +1,9 @@
+1.5.0
+=====
+
+* Replace `String` to `Text` in `LoggerName`
+* Rename `LoggerName` field name to `getLoggerName`
```

Comment:
Also you forgot to mention that you renamed `getLoggerName` inside `HasLoggerName` to `askLoggerName`.